src/content/docs/how-to/develop-apps/color-eyre/index.md

```diff
@@ -0,0 +1,102 @@
+---
+title: How to use color_eyre with Ratatui
+---
+
+Full source code for this how to article is available at:
+<https://github.com/ratatui-org/website/tree/main/code/how-to-color_eyre/>
+
+The [`color_eyre`] crate provides error report handlers for panics and errors. It displays the
+reports formatted and in color. To use these handlers, a Ratatui app needs to restore the terminal
+before displaying the errors.
+
+## Installation
+
+First add the crate to your `Cargo.toml`
+
+```shell title="add color_eyre to Cargo.toml"
+cargo add color_eyre
+```
+
+Add the following imports to `main.rs`
+
+```rust title="main.rs"
+{{ #include @code/how-to-color_eyre/src/main.rs:imports }}
+```
```

Comment:
I believe expressive code ( the functionality used by starlight ) has a feature where if you write the following:

````markdown
```rust
// main.rs
{{ #include @code/how-to-color_eyre/src/main.rs:imports }}
```
````

it does the equivalent of what you have here. This only works if the comment regex matches for a filename like string.

This has a minor advantage of the source markdown containing the filename as well.